pkg/platform/packet/template.go

```diff
@@ -133,8 +133,24 @@ module "worker-{{ $pool.Name }}" {
 
   ssh_keys  = {{$.SSHPublicKeys}}
 
+  {{- if $pool.CLCSnippets }}
+  clc_snippets = [
+      {{- range $pool.CLCSnippets }}
+      "{{ . }}",
```

Comment:
Ditto CLC snippets. For AWS, we encode them into JSON, so the `<<EOF` is not needed there.